content/docs/2.9/operate/cluster.md

```diff
@@ -75,6 +75,23 @@ $env:KEDA_HTTP_DEFAULT_TIMEOUT=1000
 
 All applicable scalers will use this timeout. Setting a per-scaler timeout is currently unsupported.
 
+## HTTP connection disable keep alive
+
+Keep alive behaviour is enabled by default for every HTTP connection, this could stack a huge amount of connections (one per scaler) in some scenarios. 
+
+You can override this default setting `KEDA_HTTP_DISABLE_KEEP_ALIVE` environment variable to true. For example, on Linux/Mac/Windows WSL2 operating systems, you'd use this command to set to true:
+
+```shell
+export KEDA_HTTP_DISABLE_KEEP_ALIVE=true
```

Comment:
We should point users here to env variables in Kubernetes Deployments not on their local machines, similar way it is documented for `HTTP Proxies` section below. 

I see that you took the reference from `HTTP Timeouts` section, which is misleading and should be changed the same way, could you please change that as well?

Reply:
@JorTurFer, I have change it, please review again, thanks